common/docs/SIE_ACS_Verification.md

```diff
@@ -85,7 +97,7 @@ truncate -s 64M flash1.img
 QEMU=<path to QEMU model>
 FLASH0=<path to flash0.img>
 FLASH1=<path to flash1.img>
-IMG=<path to systemready IR/ES/SR image>
+IMG=<path to systemready ES/S/IR image>
```

Comment:
SR
Remove IR, as it is not a general use case

IR image also be run if the firmware is UEFI using the above steps.

Reply:
Added a note.